styleguide/source/assets/scss/01-atoms/_listicle.scss

```diff
@@ -0,0 +1,52 @@
+ol.listicle {
+  counter-reset: ama-listicle;
+  list-style: none;
+  @include gutter($padding-top-half...);
+  @include gutter($padding-bottom-half...);
+
+  > li {
+
+    counter-increment: ama-listicle;
+    position: relative;
+    @include gutter($margin-bottom-half...);
+    @include gutter($margin-top-half...);
+
+    &::before {
```

Comment:
the properties for the purple box should be

` {
  width: 21px;
  height: 19px;
  font-size: 22px;
  font-weight: 600;
  letter-spacing: -1.78px;
}`